Remove unavailable commands

schierlm opened the pull request:
Closes #43

ChickenNuggers commented:
Did you make sure that this list is compatible with https://github.com/hashbang/shell-etc/issues/43#issuecomment-168362794 ?

schierlm commented:
I re-checked available binaries after he made that comment, and tried not to remove those that got installed meanwhile

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -56,24 +53,15 @@ applications written in javascript.
 sbcl - (Steel Bank Common Lisp) A Lisp implementation that features a high
 performance native compiler, Unicode support, and threading.
 
-ghc - (The Glorious Glasgow Haskell Compilation System) a native code compiler
```
ghc is installed.

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -38,9 +38,6 @@ code readability.
 
 ruby - A dynamic, object-oriented general-purpose programming language.
 
-haskell - A standardized, general-purpose programming language with non-strict
```
ghc is available.

schierlm replied:
```
mihi@da1:~$ ghc
Could not find the database of available applications, run update-command-not-found as root to fix this
ghc: command not found
```
According to https://packages.debian.org/stable/i386/ghc/filelist, the binary should be called `/usr/bin/ghc`

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -83,51 +71,21 @@ pico - (Pine Composer) a text editor originally integrated with the pine e-mail
 
 shasum - Calculates and verifies SHA hashes.
-
-bcrypt - A key derivation function for passwords based on the Blowfish cipher.
```
hrm, I thought I installed this.

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -144,112 +102,22 @@ includes many features of the C shell.
-finch - A console-based instant messaging client based on the libpurple
-library.
-
-bitlbee - Bitlbee brings Instant Messaging to IRC clients. It has support for 
```
bitlbee is installed but not running.

KellerFuchs replied:
Couldn't find a package for `centerim`.

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -144,112 +102,22 @@ includes many features of the C shell.
 
 w3m - A text based web browser and pager.
-
-html2text - Reads an HTML document and outputs plain text characters.
```
I thought I installed this.

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -144,112 +102,22 @@ includes many features of the C shell.
-mongodb - A document-oriented NoSQL database that uses JSON-like documents with
-dynamic schemas.
-
-redis - A networked, in-memory, key-value data store with optional durability
```
redis is installed.

schierlm replied:
how to launch?

daurnimator replied:
server or client or sentinel?
but, e.g. `redis-server`

schierlm replied:
you win. I think I tried to tab-complete `redis` and it found nothing, but now it does indeed find them :)

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -283,94 +141,28 @@ locate - Searches a prebuilt database for files on a filesystem.
-
-ipcalc - A program that calculates IP information for a host.
-
-socat - (SOcket CAT) A command line based utility that establishes two
```
I thought I installed this too....

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -378,7 +170,7 @@ client-server architecture.
 svn - (Subversion) A software versioning and revision control system
 maintained by apache and designed as a successor to CVS
 
-mercurial - A distributed revision control system designed for high
```
why the rename?

schierlm replied:
It seemed to me that the manpage tries to write the programs like you invoke them. Mercurial is invoked as `hg` and there is no symlink. If that is not the case, I can add some comment that it is invoked as `hg` :)

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -386,16 +178,10 @@ integrity, and support for distributed, non-linear workflows.
 
 tig - A text-mode interface for git.
 
-cloc - Counts and computes differences of lines of source code and comments.
```
sigh.. (thought it was installed)

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -386,16 +178,10 @@ integrity, and support for distributed, non-linear workflows.
 
-vimpager - Allows vim to be used as a pager for things like man pages.
-
-hub - Wrapper around git(1) that allows better integration with Github.
```
we should install this.

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -472,51 +212,14 @@ user to access multiple separate terminal sessions inside a single terminal.
-management.
-
 .SS Misc. / Unsorted (Sort these!)
-pv - Monitors the progress of data through a pipe.
```
thought this was installed.... apparnelty not. lets install.

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -472,51 +212,14 @@ user to access multiple separate terminal sessions inside a single terminal.
-
 watch - Executes a program periodically, showing the output fullscreen.
-
-libev - A high-performance event loop for C.
```
installed.

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -472,51 +212,14 @@ user to access multiple separate terminal sessions inside a single terminal.
-
-libev - A high-performance event loop for C.
-
-libevent - Provides a mechanism to execute a callback function when a specific
```
installed.

daurnimator commented on man/man7/hashbang.7:
```diff
@@ -472,51 +212,14 @@ user to access multiple separate terminal sessions inside a single terminal.
-
-cowsay - Generates ASCII pictures of a cow with a message.
-
-dos2unix - Converts line breaks in a text file from DOS format to Unix format.
```
thought I installed this....

schierlm commented:
Tell me when you believe you are finished installing and I'll update the pull request :)

schierlm commented:
Or I'll just close it.

KellerFuchs commented:
@schierlm I think I fixed today the install issues @daurnimator had, so this should finally get handled.

KellerFuchs commented:
Going towards solving this definitely: I solved the “disappearing packages” issue, and I (re)installed the following packages:

```
aria2
bcrypt
bitlbee
cloc
clojure1.6
cowsay
dos2unix
finch
ghc
golang-go
html2text
imagemagick
ipcalc
pass
pv
rtorrent
socat
```

schierlm commented:
Ok, went over the diff again and updated it

- Added back what got installed
- If the binary and the program name differ, add the binary in square brackets

I hope I did not miss anything.

KellerFuchs commented:
Ok, installed the following additional packages which seem universally useful:
```
encfs remind wyrd tudu dc qalc duplicity ranger atool p7zip zpaq nmap mtr iperf lftp zangband
 slashem frotz byobu tsung silversearcher-ag{,-el}
```

`silversearcher-ag{,-el}` are `ag` and it's Emacs integration.

KellerFuchs commented:
Aside from that, I checked your diff and it seems ok.
I will happily merge this once you add the needed packages.

schierlm commented:
Added everything except `iperf` which hasn't been on the previous manpage either.
